app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -150,14 +161,32 @@ def lt_map_set_new_form_vars
     session[:edit] = @edit
   end
 
+  def categories_for_select
+    categories = Classification.categories.sort_by(&:description)
+    categories.each_with_object([]) do |category, result|
+      next if category.read_only?
+
+      result << category.description
+    end
+  end
+
   # AJAX driven routine to check for changes in ANY field on the user form
   def label_tag_mapping_field_changed
     return unless load_edit("label_tag_mapping_edit__#{params[:id]}", "replace_cell__explorer")
 
     lt_map_get_form_vars
     @changed = (@edit[:new] != @edit[:current])
+
+    controller_object = self
+
     render :update do |page|
       page << javascript_prologue
+
+      if params[:entity]
+        page.replace("tag_category_div", :partial => "ops/label_tag_mapping/tag_category",
```

Comment:
@lpichler I'm trying to run this locally and getting this error for the missing partial -
```
[----] F, [2020-10-06T14:04:46.999003 #22945:3fed75120b90] FATAL -- : Error caught: [ActionView::MissingTemplate] Missing partial ops/label_tag_mapping/_tag_category with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :haml, :coffee, :rjs]}. Searched in:
  * "/Users/greggtanzillo/work/ibm/manageiq-ui-classic/app/views"
  * "/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/bundler/gems/manageiq-v2v-6f8e502c5e2f/app/views"

/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/actionview-5.2.4.4/lib/action_view/path_set.rb:48:in `find'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/actionview-5.2.4.4/lib/action_view/lookup_context.rb:116:in `find'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/actionview-5.2.4.4/lib/action_view/renderer/partial_renderer.rb:423:in `find_template'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/actionview-5.2.4.4/lib/action_view/renderer/partial_renderer.rb:418:in `find_partial'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/actionview-5.2.4.4/lib/action_view/renderer/partial_renderer.rb:299:in `render'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/actionview-5.2.4.4/lib/action_view/renderer/renderer.rb:49:in `render_partial'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/actionview-5.2.4.4/lib/action_view/renderer/renderer.rb:23:in `render'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/actionview-5.2.4.4/lib/action_view/helpers/rendering_helper.rb:33:in `render'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/jquery-rjs-0.1.1.1/lib/action_view/helpers/jquery_helper.rb:566:in `block in render'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/jquery-rjs-0.1.1.1/lib/action_view/helpers/jquery_helper.rb:579:in `with_formats'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/jquery-rjs-0.1.1.1/lib/action_view/helpers/jquery_helper.rb:563:in `render'
/Users/greggtanzillo/.rvm/gems/ruby-2.5.3@vmdb_latest/gems/jquery-rjs-0.1.1.1/lib/action_view/helpers/jquery_helper.rb:381:in `replace'
/Users/greggtanzillo/work/ibm/manageiq-ui-classic/app/controllers/ops_controller/settings/label_tag_mapping.rb:186:in `block in label_tag_mapping_field_changed'
```
I can't seem to find it. Is it possible that it was not pushed up tho your branch?

Reply:
@gtanzillo yes that's right I did not push haml file `ops/label_tag_mapping/tag_category. haml`
it is added now, thanks